_posts/2013-04-06-developing-addons-and-blueprints.md

```diff
@@ -138,6 +138,20 @@ Let's add some meta data to categorize the addon a little better:
 ]
 {% endhighlight %}
 
+### Addons depending on other addons
+If your addon depends on another addon, install it as a dependency in your
+`package.json`:
+
+{% highlight javascript %}
+// ...
+"dependencies": {
+  "ember-ajax": "^0.7.1"
+}
+{% endhighlight %}
+
+An application that consumes your addon will automatically install
+and bundle dependencies specified this way.
```

Comment:
Does this mean that the consuming application will be able to use these it directly as well?